src/main/java/pro/quicksense/modules/controller/SysUserController.java

```diff
@@ -0,0 +1,205 @@
+package pro.quicksense.modules.controller;
+
+
+import cn.hutool.core.util.ObjectUtil;
+import com.alibaba.fastjson.JSONObject;
+import com.baomidou.mybatisplus.core.conditions.query.LambdaQueryWrapper;
+import com.baomidou.mybatisplus.core.toolkit.StringUtils;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+import pro.quicksense.modules.common.CommonConstant;
+import pro.quicksense.modules.common.Result;
+import pro.quicksense.modules.entity.User;
+import pro.quicksense.modules.service.ISysUserService;
+import pro.quicksense.modules.util.CodeUtil;
+import pro.quicksense.modules.util.EmailUtil;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+
+@Slf4j
+@RestController
+@RequestMapping("/user")
+public class SysUserController {
+
+    @Autowired
+    private ISysUserService sysUserService;
```

Comment:
```suggestion
public class UserController {

    @Autowired
    private UserService UserService;
```
also modify the file name.